recipes/pyobjc-core/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyobjc-core" %}
+{% set version = "3.2.1" %}
+{% set md5 = "df752f952825dd68958c28a9b07967b6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/7e/03/303a5c7f7c3d3af811eba44b32ef957e570be4d5c5b656c0b44ece6191e0/{{ name }}-{{version}}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: true  # [not osx]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - PyObjCTools
+    - objc
+
+about:
+  home: http://pyobjc.sourceforge.net/
+  license: MIT
```

Comment:
Is there a license file that we can include from the source in the built package?